### PR TITLE
docs: add docs for preview environments

### PR DESCRIPTION
**Pull request by JivusAyrus**
#### Checklist

- [ ] run `make test`
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs-website/src/pages/docs/cloud/preview-environments.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'Preview Environments'
+pageTitle: WunderGraph - Preview Environments
+description: An overview of preview environments
+---
+
+Preview Environments enable you to preview your API changes without merging them to your production branch.
+
+- Preview Environments are available by default for all the projects.
+- Preview Environments will be created when a pull request is raised against the production branch.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/preview-environments.md:10]: Perhaps "A Preview Environment will be created for each pull request raised against the project's production branch."

Comment (JivusAyrus) [reply docs-website/src/pages/docs/cloud/preview-environments.md]: Done

File: docs-website/src/pages/docs/cloud/preview-environments.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'Preview Environments'
+pageTitle: WunderGraph - Preview Environments
+description: An overview of preview environments
+---
+
+Preview Environments enable you to preview your API changes without merging them to your production branch.
+
+- Preview Environments are available by default for all the projects.
```
[the remark this answers is not in the document]
Comment (JivusAyrus) [reply docs-website/src/pages/docs/cloud/preview-environments.md]: Done

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**